### DISCUSSION TITLE
google.maps.Marker is deprecated

### DISCUSSION THREAD

**Discussion by abhishekmardiya** [visgl/react-google-maps#246] [General]
I'm having this warning:
"As of February 21st, 2024, google.maps.Marker is deprecated. Please use google.maps.marker.AdvancedMarkerElement instead. Please see https://developers.google.com/maps/deprecations for deprecation details. "

![Screenshot (478)](https://github.com/visgl/react-google-maps/assets/97448460/238f7583-9834-418b-b81b-44c07d25846d)

**Comment by wangela** [comment]
The `google.maps.Marker` class still works, it's just labeled as deprecated since `google.maps.marker.AdvancedMarkerElement` works and is better. See https://developers.google.com/maps/documentation/javascript/advanced-markers/migration for tips on migrating from Marker to AdvancedMarkerElement, or take a look at the docs for [`<Marker>`](https://visgl.github.io/react-google-maps/docs/api-reference/components/marker) vs [`<AdvancedMarker>`](https://visgl.github.io/react-google-maps/docs/api-reference/components/advanced-marker).

**Reply by abhishekmardiya** [reply]
Thank you!
